components/telegram_bot_api/telegram_bot_api.app.js

```diff
@@ -58,6 +74,161 @@ module.exports = {
+      optional: true,
+    },
+    reply_to_message_id: {
+      type: "string",
+      label: "Original Message ID",
+      description: "Enter the ID of the original message.",
+      optional: true,
+    },
+    reply_markup: {
+      type: "string",
+      label: "Reply Markup",
+      description: "Enter additional interface options that are a JSON-serialized object including an inline keyboard, a custom reply keyboard, instructions to remove the reply keyboard or instructions to force a reply from the user, e.g. `{\"inline_keyboard\":[[{\"text\":\"Some button text 2\",\"url\":\"https://botpress.org\"}]]}` or `{\"keyboard\":[[\"Yes\",\"No\"],[\"Maybe\"]]}`. Note: keyboard cannot be used with channels.",
+      optional: true,
+    },
+    messageId: {
+      type: "string",
+      label: "Message ID",
+      description: "Enter the message ID.",
+    },
+    fromChatId: {
+      type: "string",
+      label: "From Chat ID",
+      description: "Enter the unique identifier for the chat where the original message was sent (or channel username in the format @channelusername).",
+      optional: true,
+    },
+    caption: {
+      type: "string",
+      label: "Caption",
+      description: "Enter the caption.",
+      optional: true,
+    },
+    filename: {
+      type: "string",
+      label: "File Name",
+      description: "Enter a filename.",
+      optional: true,
+    },
+    media: {
+      type: "string",
+      label: "Media File Source",
+      description: "File to send. Pass a file_id to send a file that exists on the Telegram servers, pass an HTTP URL for Telegram to get a file from the Internet, or pass the path to the file (e.g., `/tmp/myFile.ext`) to upload a new one using a file [downloaded to `/tmp`](https://pipedream.com/docs/workflows/steps/code/nodejs/working-with-files/#download-a-file-to-tmp). File must meet Telegram's [requirements](https://core.telegram.org/bots/api#sending-files) for MIME type and size.",
+    },
+    duration: {
+      type: "integer",
+      label: "Duration",
+      description: "Enter duration of sent video in seconds.",
+      optional: true,
+    },
+    performer: {
+      type: "string",
+      label: "Performer",
+      description: "Enter a performer.",
+      optional: true,
+    },
+    title: {
+      type: "string",
+      label: "Title",
+      description: "Enter a track name.",
+      optional: true,
+    },
+    contentType: {
+      type: "string",
+      label: "Content Type",
+      description: "Select or enter the MIME type of data.",
+      optional: true,
+    },
+    width: {
+      type: "integer",
+      label: "Width",
+      description: "Enter the video width.",
+      optional: true,
+    },
+    height: {
+      type: "integer",
+      label: "Height",
+      description: "Enter the video height.",
+      optional: true,
+    },
+    length: {
+      type: "integer",
+      label: "Length",
+      description: "Enter the video width and height.",
```

Comment:
I didn't get this description, is it the duration of a video?

Reply:
Updated the `description` of `length`:

https://github.com/PipedreamHQ/pipedream/blob/c80b5afcf9851debaa7cab8efab46a3619aeaa1f/components/telegram_bot_api/telegram_bot_api.app.mjs#L168